scripts/release.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env sh
+set -e
+PATH=$PATH:$PWD/bin
+GOPATH=$PWD
+
+SRC=${PWD}/src/github.com/SUSE/cf-plugin-backup
+TAG=$(git -C ${SRC} describe --tags --long | awk -F - '{ print $1 }' )
+NAME=$(basename $(git -C ${SRC} config --get remote.origin.url) .git)
+
+# Assemble configuration data for `out` to git-release resource.
+echo > release/name ${TAG}
+echo > release/tag  ${TAG}
+echo > release/body "${NAME}: Release ${TAG}"
```

Comment:
No need to do it in this PR, but a nicer template would be nice in the draft. Something like what you have, plus the output of `git log --reverse --pretty=%s --merges $(git describe --tags --abbrev=0 HEAD^)...${TAG}`.

Reply:
will try, just to see how it looks

Reply:
Done with commit 26ec7b3de6826d8677420fe18dc7d72b1f188c88